README.md

```diff
@@ -26,12 +26,8 @@ Have a look at [`test/probes`](./test/probes) directory to see how you can use t
 defmodule MyApp.Router do
   use Phoenix.Router
 
-  forward "/alive", Healthchex.Probes.Liveness
-  forward(
-    "/ready",
-    Healthchex.Probes.Readiness,
-    probe: &Domain.db_ready?/0
-  )
+  forward "/health/live", Healthchex.Probes.Liveness
+  forward "/health/ready", Healthchex.Probes.Readiness
```

Comment:
```suggestion
  forward(
    "/health/ready",
    Healthchex.Probes.Readiness,
    probe: &Domain.db_ready?/0
  )
```

I wanted to show here that we can provide our own `probe` function.

Reply:
I'll had a section about that in the README.MD

I was just thinking it would be nice for the first example to be the very basic use case, that someone could copy and paste directly into their code.